packagehandlers/npmpackagehandler.go

```diff
@@ -19,5 +29,14 @@ func (npm *NpmPackageHandler) UpdateDependency(vulnDetails *utils.VulnerabilityD
 }
 
 func (npm *NpmPackageHandler) updateDirectDependency(vulnDetails *utils.VulnerabilityDetails) (err error) {
-	return npm.CommonPackageHandler.UpdateDependency(vulnDetails, vulnDetails.Technology.GetPackageInstallationCommand())
+	isNodeModulesExists, err := fileutils.IsDirExists(filepath.Join(".", "node_modules"), false)
```

Comment:
I think that you'll get the same results with this:
```suggestion
	isNodeModulesExists, err := fileutils.IsDirExists("node_modules", false)
```